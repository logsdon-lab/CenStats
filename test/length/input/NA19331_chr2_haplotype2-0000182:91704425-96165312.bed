version https://git-lfs.github.com/spec/v1
oid sha256:779a4906b1f45a5a70fa35d6e7561cb987d4c83c1d30bff3c0bbfdf41a0d3eff
size 740066
