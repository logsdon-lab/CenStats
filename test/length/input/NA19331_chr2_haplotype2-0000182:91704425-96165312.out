version https://git-lfs.github.com/spec/v1
oid sha256:3525f0faa5ab66cd923971e0307cfe411be7c95b0dcb50c6b60bdaffa8faa957
size 205185
